src/map/skill.cpp

```diff
@@ -5583,42 +5583,44 @@ int32 skill_castend_damage_id (struct block_list* src, struct block_list *bl, ui
 
 	case RG_BACKSTAP:
 		{
-			if (!check_distance_bl(src, bl, 0)) {
+#ifndef RENEWAL
+		if (check_distance_bl(src, bl, 0))
+			break;
```

Comment:
Hmm, now that I look at it, might be better to just move this code into the "else" part so that you don't need to ifdef twice.

Reply:
Also you removed two tabs instead of just one. So there no indent after the first { here and the last } is on the same height as the code, which makes it a bit confusing.

Reply:
Done fixing indent and no more 2 IFDEF.